website/docs/r/tag_category.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_tag_category"
+sidebar_current: "docs-vsphere-resource-inventory-tag-category"
+description: |-
+  Provides a vSphere tag category resource. This can be used to manage tag categories in vSphere.
+---
+
+# vsphere\_tag\_category
+
+The `vsphere_tag_category` resource can be used to create and manage tag
+categories, which determine how tags are grouped together and applied to
+specific objects.
+
+For more information about tags, click [here][ext-tags-general]. For more
+information about tag categories specifically, click
+[here][ext-tag-categories].
+
+[ext-tags-general]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html
+[ext-tag-categories]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vcenterhost.doc/GUID-BA3D1794-28F2-43F3-BCE9-3964CB207FB6.html
+
+~> **NOTE:** Tagging support is unsupported on direct ESXi connections and
```

Comment:
this probably also warrants a section on the main vSphere page for "Required vSphere Versions" IMO (although, [checking the page](https://www.terraform.io/docs/providers/vsphere/index.html) perhaps it wants a separate page for that?)

Reply:
Honestly that index page needs a massive overhaul anyway so I'm not surprised. For now I think I'm fine with it being out of date, but close to v1.0.0 (or during some downtime) I will work on updating it so that not only does it have this information, but the dev info is removed, and other steps are taken to make sure that it focuses on a summary of what the provider does and that's it, saving extra detail for each individual resource.